prow/gerrit/client/client_test.go

```diff
@@ -99,6 +99,35 @@ func TestQueryChange(t *testing.T) {
 			},
 			revisions: map[string][]string{},
 		},
+		{
+			name:       "one outdated change, but there's a new message",
+			lastUpdate: now,
```

Comment:
tiny nit: I'll suggest use `lastUpdate: now.Add(-time.Minute)` here, and `Updated: now.Format(layout)` below, to avoid test things in the future.

Reply:
done